File: java/src/main/java/gov/doc/ntia/sigmf/ext/annotation/environment/SensorEnvironment.java
package gov.doc.ntia.sigmf.ext.annotation.environment;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class SensorEnvironment extends Environment  {

    private static final long serialVersionUID = 1L;

    @JsonProperty(value="ntia-environment:category", required = false)
    protected String category;


    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }




}

File: Csharp/sigmf/sigmf/ext/Annotationdir/environment/SensorEnvironment.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace sigmf
{
    [Serializable()]
    public class SensorEnvironment : Environment
    {
        private static readonly long serialVersionUID = 1L;

        [JsonProperty("ntia-environment:category")]
        protected string category1;

        public string Category1 { get => category1; set => category1 = value; }
    }
}
